test/unittest/tests/wrappers/config_parser.h

```diff
@@ -0,0 +1,27 @@
+/*-
+ * Copyright (c) 2014-present MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+#pragma once
+
+#include "wt_internal.h"
+#include <map>
+#include <string>
+
+class config_parser {
+public:
+    config_parser(std::map<std::string, std::string> map);
+    ~config_parser() = default;
+
+    std::map<std::string, std::string> &get_config_map();
```

Comment:
Adding a `const` version of `get_config_map` allows us to pass configs as `const&` parameters in more places:

```
    std::map<std::string, std::string> &get_config_map();
    std::map<std::string, std::string> const& get_config_map() const;
```